src/System.Azure.Experimental/System/Azure/CosmosDbAuthorizationHeader.cs

```diff
@@ -42,7 +42,7 @@ public static bool TryWrite(Span<byte> output, Sha256 hash, string keyType, stri
             writer.Write(tokenVersion);
             writer.Write("&sig=");
 
-            writer.WriteBytes(hash, default, Base64.BytesToUtf8Encoder);
+            writer.WriteBytes(hash, default, Base64Experimental.BytesToUtf8Encoder);
```

Comment:
Why Base64Experimental? Why not call the corfxlab APIs?

Reply:
I don't understand what you mean. `Base64Experimental.BytesToUtf8Encoder` is a type in corefxlab.
https://github.com/dotnet/corefxlab/pull/1896/files#diff-bdbcc9558ff685630e7887fe9c35bf95R14

The WriteBytes method takes an IBufferTransformation as argument so that is why I am passing a Utf8Encoder to it.

Reply:
Why do we need Base64Experimental give now we have official Base64 APIs in corfx.

Reply:
For the types that implement IBufferTransformation and IBufferOperation such as Utf8Encoder and Utf8Decoder.
https://github.com/ahsonkhan/corefxlab/blob/master/src/System.Binary.Base64/System/Binary/Base64Encoder.cs#L207

Reply:
Ah, thanks. We might need to come up with a better name/design for these, but for now it's ok.